lms/templates/dashboard.html

```diff
@@ -58,6 +126,11 @@
         banner.showMessage(${json.dumps(redirect_message)})
     </%static:require_module>
   % endif
+  % if settings.FEATURES.get('ENABLE_TABBED_DASHBOARD'):
```

Comment:
Instead of calling settings in templates, pass a context variable (e.g. `enable_tabbed_dashboard`). Mako makes it too easy to violate MVC/MVT.